kratos/tests/cpp_tests/strategies/strategies/test_explicit_strategies.cpp

```diff
@@ -50,7 +50,7 @@ namespace Testing
 
         AuxiliaryExplicitStrategiesTestElement(AuxiliaryExplicitStrateguesTestElement const& rOther) = delete;
 
-        ~AuxiliaryExplicitStrateguesTestElement() = default;
+        ~AuxiliaryExplicitStrategiesTestElement() = default;
```

Comment:
shouldn't this be virtual?

Reply:
no its not a baseclass

but probably an override is missing

Reply:
As doing `override = default` is equivalent to say that this destructor does nothing, I'd remove it.